### PR TITLE
Cleanup warnings and CodeQL issues

### PR DESCRIPTION
**Pull request by jas88**
Cleaning up CodeQL and ReSharper issues

### CODE REVIEW

**Comment by jas88** [comment]
(57 CodeQL issues fixed in this PR)

File: IsIdentifiable/Whitelists/CsvWhitelist.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using CsvHelper;
 using CsvHelper.Configuration;
 
-namespace IsIdentifiable.Allowlists;
+namespace IsIdentifiable.Whitelists;
```
Comment (rkm) [inline IsIdentifiable/Whitelists/CsvWhitelist.cs:7]: What's the reason for the change here? I think we should prefer allowlist over whitelist

**Review by rkm — CHANGES_REQUESTED** [review]

File: IsIdentifiable/Whitelists/CsvWhitelist.cs (hunk 1 above)
Comment (jas88) [reply IsIdentifiable/Whitelists/CsvWhitelist.cs:7]: Agreed - the namespace inside the file said "allowlists" but the actual filename was still "whitelists" so the discrepancy got flagged.

**Review by rkm — APPROVED** [review]
